HeterogeneousCore/CUDAUtilities/interface/launch.h

```diff
@@ -94,10 +94,14 @@ namespace cudautils {
   }  // namespace detail
 
   // wrappers for cudaLaunchKernel
-
+  inline
```

Comment:
this was missing in the original implementation

Reply:
do you mean the `inline` ?
thanks for fixing

Reply:
yes, the two missing inline